.env.schema

```diff
@@ -16,4 +16,10 @@ REACT_APP_DEBUG=app
 
 # arranger values
 REACT_APP_ARRANGER_API=http://...
-REACT_APP_PROJECT_ID=
\ No newline at end of file
+REACT_APP_PROJECT_ID=
```

Comment:
can you help me add `REACT_APP_GOOGLE_MAPS_KEY` to this file while you are here 🙏